lib/easypost/shipment.rb

```diff
@@ -6,6 +6,12 @@ def get_rates(params={})
     return self
   end
 
+  def get_smartrates
+    response = EasyPost.make_request(:get, url + '/smartrate', @api_key)
+
+    return response['result']
```

Comment:
Do we want to use the safe navigation operator, `&.` or a `dig` ? For example, this would be useful if the API was down, or if the API key is incorrect, etc.

Reply:
Using `fetch` here with a fallback value of an empty array.